Articles/ShortArticles/PullRequestSizeConsiderations.md

```diff
@@ -157,9 +157,9 @@ for large and/or automated refactorings.
 [26]: https://news.ycombinator.com/item?id=13561096 " Hacker News commentary on Google's Billion Lines of Code Repository {}"
 
 <!---
- Publish: no
+ Publish: preview
```

Comment:
@rinkug, no, we don't want to view the source file, only the generated file.

@markcmiller86, @bernhold ?

Reply:
I think the only constraint here is that the *source* should never be set to *publish*.  I think no or preview are equally fine.  With no, of course, it will not even be visible in the preview site.  If a contributor wants to see the source in preview, I don't see any problem with that.

I think I set it to no while debugging the title collision, hoping that it would eliminate the problem.  Unfortunately, it did not.